src/main/java/smokefree/domain/IndicatePlaygroundObservationCommand.java

```diff
@@ -8,11 +8,10 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 @AllArgsConstructor
 @ToString
-public class RecordSmokeFreePlaygroundObservationCommand {
+public class IndicatePlaygroundObservationCommand {
     @TargetAggregateIdentifier
     String initiativeId;
-    String citizenId;
```

Comment:
I would keep the citizenId, but instead name it 'observer'. 
I know that you can take the actual id from the authentication metadata and I have been thinking about this. I think that the commands should be a "complete" record and that the metadata should be used to validate. (In this case we need to validate that the observer === the authenticated user). It may seem superfluous, but what if in the future somebody (a moderator perhaps) should be able to issue command on behalf of somebody else? We need to document these guidelines of course, but for now this is the direction that I propose. If you think differently, let's discuss.

Reply:
Yeah, Sounds good. Yes, It's possible If a moderator has given such options to comment on behalf of volunteers.